app/view/scorecard-form/index.js

```diff
@@ -0,0 +1,116 @@
+'use strict';
+require('./scorecard-form.scss');
+
+const angular = require('angular');
+const appShooter = angular.module('appShooter');
+
+appShooter.controller('CreateScorecardFormController', ['$log','$scope', '$location', 'scorecardService', CreateScorecardFormController]);
+
+function CreateScorecardFormController($log, $scope, $location, scorecardService){
+  $log.debug('entered CreateScorecardFormController');
+  const vm = this;
```

Comment:
great use of angular best pratices,
however you should remember to be consistant through out your code base
eaither use this, or vm = this, but not both in one code base